### ISSUE TITLE
Add docs for Dictionaries methods

### ISSUE THREAD

**Issue by kescobo** [BioJulia/Microbiome.jl#76]
`set!`, `unset!`, `insert!`, `delete!`, `haskey!`, and `keys` methods for both `MicrobiomeSample` and `CommunityProfile` all need docstrings. They can be pretty similar. Can be very similar to the ones in `Dictionaries` itself:

```
julia> using Dictionaries

help?> set!
search: set! unset! insert! intersect! setdiff! setfield! setindex! sizehint! setindices! setproperty! unsafe_store! unsafe_copyto!

  set!(dict::AbstractDictionary, i, value)


  Update or insert the value at index i into dict. Sometimes referred to as an "upsert" operation.

  Hint: Use setindex! to exclusively update an existing value, and insert! to exclusively insert a new value. See also get!.

  ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

  set!(indices::AbstractIndices, i)


  Insert a new value i into indices if it doesn't exist, or do nothing otherwise.
```

cc @anikaluo @annelle-abatoni

**Comment by annelle-abatoni** [comment]
I added these docstrings for CP, but I am not able to push them because I'm not listed as a collaborator, although I accepted the invitation you recently sent out. I get this error:

Warning: Permanently added the RSA host key for IP address '140.82.114.3' to the list of known hosts.
ERROR: Permission to Biojulia/Microbiome.jl.git denied to annelle-abatoni.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights and the repository exists.

Help with this, please!

**Comment by kescobo** [comment]
Please make a fork, then push to a branch on that fork, then make a pull request

**Comment by kescobo** [comment]
Closed by #82
